apps/examples.nameguard.io/postcss.config.cjs

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   plugins: {
     tailwindcss: {},
-    autoprefixer: {},
-  },
-}
+    autoprefixer: {}
+  }
+}
```

Comment:
Blank line needed

Reply:
Interesting. What creates this need? I added one as requested but don't see what it changes. If some different fix is needed please feel welcome to action that directly in this PR :+1: